ansible/roles/rancher/tasks/rancher.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Add Rancher Repo
+  command: helm repo add rancher-stable https://releases.rancher.com/server-charts/stable
+  when: "'rancher-stable' not in repos.stdout"
+
+- name: Install Rancher
+  command: helm install rancher-stable/rancher --name rancher --namespace cattle-system --version 2.2.8 --set tls=external --set privateCA=true --set addLocal=false --set auditLog.level=1 --description='RanchHand Deploy' --wait
```

Comment:
How would we apply updates to Rancher itself and, separately, Kubernetes?

Reply:
We should be able to just specify the `--version` flag to point to the desired upgrade version and Helm should carry out the rest of the rancher upgrade management.